src/spikeinterface/comparison/groundtruthstudy.py

```diff
@@ -185,6 +185,9 @@ def run_sorters(self, case_keys=None, engine="loop", engine_kwargs={}, keep=True
                 if log_file.exists():
                     log_file.unlink()
 
+            if sorter_folder_exists:
```

Comment:
this is not correct. You have an issue L168 also, with wrong names

Reply:
The if L181 should be if sorter_folder_exists no?

Reply:
L181 is OK we are removing the sorting folder